src/index.html

```diff
@@ -3,11 +3,219 @@
   <head>
     <meta charset="UTF-8">
     <title>Title</title>
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <link rel="stylesheet" href="./styles/main.css">
+    <link href="https://fonts.googleapis.com/css?family=Roboto:400,500,700&display=swap" rel="stylesheet">
+
   </head>
   <body>
-    <img src="./images/logo.svg" alt="Logo">
-    <h1>Hello Mate Academy</h1>
+
+    <header class="header">
+      <div class="wrapper wrapper-header-items">
+        <a href="#">
+          <img class="header__logo" src="./images/logo.png" alt="uber logo">
+        </a>
+        <div class="header-wrapp-input large-size-inputs">
+          <input class="header__input header__input--when" type="text" name="when" placeholder="ASAP">
+          <span class="header__span">to</span>
+          <input class="header__input header__input--to" type="text" name="to" placeholder="ul. Bolshaya Dmitrov">
+        </div>
+        <div class="header-wrapp-buttons">
+          <button class="header__button header__button--signin" type="button" name="button">
+            <a class="signin-link" href="#">Sign in</a>
+          </button>
+          <button class="header__button header__button--register" type="button" name="button">
+            <a class="register-link" href="#">Register</a>
+          </button>
+          <a class="header__imgbasket" href="#">
+            <img src="./images/basket.png" alt="basket image">
+          </a>
+        </div>
+      </div>
+    </header>
+
+    <main class="main">
+      <div class="wrapper wrapper-main-items">
+        <input class="main__search" type="search" name="searchproducts" placeholder="Search for restaurant or cuisine">
```

Comment:
Делай переносы слишком длинных тегов
```suggestion
        <input
          class="main__search"
          type="search"
          name="searchproducts"
          placeholder="Search for restaurant or cuisine"
        >
```